wagtail_purge/wagtail_hooks.py

```diff
@@ -14,9 +14,14 @@ def register_admin_urls():
 
 @hooks.register("register_settings_menu_item")
 def register_purge_menu_item():
+    if WAGTAIL_VERSION >= (5, 2):
+            kwargs = {"classname": "icon icon-collapse-down"}
+        else:
```

Comment:
It looks like this is at the wrong indentation level relative to the if-statement :)

Reply:
Otherwise though, I like this check

Reply:
Thanks, didn't see this. Please see latest commit.